debian/patches/pang12.patch

```diff
@@ -0,0 +1,16 @@
+Index: systemd/hwdb.d/60-keyboard.hwdb
+===================================================================
+--- systemd.orig/hwdb.d/60-keyboard.hwdb
++++ systemd/hwdb.d/60-keyboard.hwdb
+@@ -1693,6 +1693,11 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnSystem7
+  KEYBOARD_KEY_f7=f21                                    # Touchpad toggle
+  KEYBOARD_KEY_f8=f21                                    # Touchpad toggle
+ 
++# Pangolin 12
++evdev:atkbd:dmi:bvn*:bvr*:bd*:svnSystem76*:pnPangolin*:pvrpang12*
++ KEYBOARD_KEY_76=f21                                    # Touchpad toggle
++ KEYBOARD_KEY_99=prog1                                  # Fn+End; Programmable
```

Comment:
The Fn+End next song mapping seems intentional. Fn+Page Up is mapped to stop and Fn+Page Down is mapped to previous song. I'd leave those as-is.

Reply:
Ah ok, strange we didn't label that on the keyboard. Fixed in https://github.com/pop-os/systemd/pull/29 to also fix build